__tests__/app.test.js

```diff
@@ -91,3 +84,121 @@ describe("GET /api/", () => {
 
 
 
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+describe("GET /api/articles", () => {
+    test("check the request returns a 200 and has the correct properties", () => {
+
+        return request(app)
+        .get("/api/articles")
+        .expect(200)
+        .then(({body}) => {
+
+        expect(body).toHaveLength(13)    
+        body.forEach(article => {
+        expect(article).toHaveProperty('author', expect.any(String))
+        expect(article).toHaveProperty('title', expect.any(String))
+        expect(article).toHaveProperty('article_id', expect.any(Number))
+        expect(article).toHaveProperty('topic', expect.any(String))
+        expect(article).toHaveProperty('created_at', expect.any(String))
+        expect(article).toHaveProperty('votes', expect.any(Number))
+        expect(article).toHaveProperty('article_img_url', expect.any(String))
+        expect(article).toHaveProperty('commentCount', expect.any(Number))
+        expect(article).not.toHaveProperty('body')
+
+         })
+
+        })
+
+    })
+
+    test("check the request returns in date descending order", () => {
+        return request(app)
+        .get("/api/articles")
+        .expect(200)
+        .then((body) => {
+
+        expect([body._body[0].created_at , body._body[1].created_at,body._body[5].created_at, body._body[9].created_at ]).toBeSorted({descending: true})
+
```

Comment:
This is a really confusing expect statement -could you change it to something simpler? Perhaps just testing the array?